### ISSUE TITLE
Add all AP items

### ISSUE THREAD

**Issue by wookiefriseur** [manavortex/FurnitureCatalogue#67]
Currently we have the Tomato variants of PvP stuff. Add Smurfs and Bananas to improve search results.

**Comment by wolfstar07** [comment]
I completed all these. Since they were in rumor anyway, it made sense to just move them over to PVP. Can't count on people to read the addon page and see the note about the faction furnishings all being the same price.

**Comment by manavortex** [comment]
No, you can't trust people to read. Ever. Exhibit a: Nexusmods 😏
